### PR TITLE
Issue 90 fancybox

### PR DESCRIPTION
**Pull request by ledenev-viktor**
added fancybox
pull request to discuss improvements

### CODE REVIEW

**Comment by ledenev-viktor** [comment]
It turned out that it is impossible to get a full-size image using a regular link from Google Drive. The link for the full-length image is generated only for downloading. Therefore, I had to add a field for the base64 image.
Base64 is registered in the database. Not sure if it's worth doing this. It might be better to code through a Filereader on the front-end.

Among the additional functionality for the admin panel of the main slider, you can add sorting of slides through drag-and-drop.

File: src/components/ui/admin/settings/listSlides/item.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                    {/* TODO: convert to Image Nextjs */}
+                    <img
+                        src={slide.img || '/logo.png'}
+                        width={150}
+                        height={80}
+                        alt={slide.title || ''}
+                    />
+                </div>
+                <Flex className="list-inputs" vertical>
+                    <div>
```
Comment (YuliaKletsova) [inline src/components/ui/admin/settings/listSlides/item.tsx:43]: Looks like this div can be removed

File: src/components/widgets/home/mockSlides.ts
Comment (YuliaKletsova) [inline src/components/widgets/home/mockSlides.ts:1]: Can we remove this?

File: public/parallax-gallery/1.webp
Comment (YuliaKletsova) [inline public/parallax-gallery/1.webp:1]: This folder also can be removed

**Review by YuliaKletsova — CHANGES_REQUESTED** [review]

File: pages/api/supabase/settings_preview/get.tsx
Comment (YuliaKletsova) [inline pages/api/supabase/settings_preview/get.tsx:1]: Let's call the folder just settings supabase/settings/get

File: src/components/ui/admin/settings/listSlides/item.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Flex, Input, AutoComplete, Checkbox } from 'antd';
+import { useMotionValue, Reorder } from 'framer-motion';
+import { Close } from '~components/ui/icons/close';
+import { useRaisedShadow } from './hooks/useRaisedShadow';
+const { TextArea } = Input;
+
+export const Item = ({
```
Comment (YuliaKletsova) [inline src/components/ui/admin/settings/listSlides/item.tsx:8]: Could we have better naming and call the file Item? And please call the file exactly like component Item

File: src/components/ui/admin/settings/listSlides/types/index.ts
Comment (YuliaKletsova) [inline src/components/ui/admin/settings/listSlides/types/index.ts:1]: This type can be in the file where you use it.

Comment (YuliaKletsova) [reply src/components/ui/admin/settings/listSlides/types/index.ts:1]: Seems like it is just one file. But if no okay.

File: src/components/ui/modal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import styled from '@emotion/styled';
+import { Modal as ModalAntd } from 'antd';
+
+export const Modal = styled(ModalAntd)``;
```
Comment (YuliaKletsova) [inline src/components/ui/modal/index.tsx:4]: There are no styles.So let's remove this file?

File: pages/api/supabase/settings_preview/get.tsx
Comment (ledenev-viktor) [reply pages/api/supabase/settings_preview/get.tsx:1]: We use two tables in our database. SETTINGS table and PREVIEW_SETTINGS table. Therefore, I suggest leaving the directories as they are.

File: src/components/ui/admin/settings/listSlides/types/index.ts
Comment (ledenev-viktor) [reply src/components/ui/admin/settings/listSlides/types/index.ts:1]: This file used in two different places.

**Pull request merged**